aws-lambda-layer/Rakefile

```diff
@@ -30,19 +31,21 @@ task :build, [:freetds_version] do |_t, args|
 end
 
 desc 'Build FreeTDS AWS Lambda Layer'
-task :deploy, [:freetds_version] => [:build] do |_t, args|
+task :deploy, [:freetds_version, :aws_region] => [:build] do |_t, args|
```

Comment:
Given that we have an env var for this, do we also need a task parameter? I think it would probably best to just have one way to control this. The current iteration sets up a weird conflict, eg what does this mean: `AWS_REGION=whatever bundle exec rake deploy[1.1.6, some_other_region]`.

I realize we also have that conflict with the freetds version parameter, but that makes it worse somehow.

---

How about this: for now, either:

1. make it work exactly like the freetds version stuff. (hard code a const, at the top, then do a 3 layer coalesce for the effective value)
2. refactor both parameters to be exclusively env vars or task params but not both

Reply:
I realize I recommended splitting the coalescing in a prior review, but that's because I didn't realize what we were doing with freetds version at the time 🤷‍♂️🤦‍♂️